anaconda_project/internal/cli/main.py

```diff
@@ -333,6 +334,7 @@ def add_package_args(preset):
     preset = subparsers.add_parser('remove-packages', help="Remove packages from one or all project environments")
     add_directory_arg(preset)
     add_env_spec_arg(preset)
+    preset.add_argument('--pip', action='store_true', help='Uninstall the requested packages using pip.')
```

Comment:
Yeah, I see what you mean here. Might take some effort due to current implementation (most of the challenge is actually re-writing the anaconda-project.yml file rather than actually uninstalling the package from the env)